features/netsocket/NetworkStack.h

```diff
@@ -116,7 +161,7 @@ class NetworkStack: public DNS {
      *  @param address  Destination for the host address
      *  @return         NSAPI_ERROR_OK on success, negative error code on failure
      */
-    virtual nsapi_error_t add_dns_server(const SocketAddress &address);
+    virtual nsapi_error_t add_dns_server(const SocketAddress &address, const char *interface_name);
```

Comment:
Should default to to = NULL, parameter description missing

Reply:
Done